pulsar/internal/connection_pool.go

```diff
@@ -46,6 +46,7 @@ type connectionPool struct {
 	maxConnectionsPerHost int32
 	roundRobinCnt         int32
 	metrics               *Metrics
+	defaultMaxMessageSize int
```

Comment:
Suggest making this an `int32` to be consistent with the type of `connection.defaultMaxMessageSize`.